benefits/admin.py

```diff
@@ -39,3 +39,7 @@ def index(self, request, extra_context=None):
                 context.update({"has_permission_for_in_person": has_permission_for_in_person})
 
             return TemplateResponse(request, "admin/agency-dashboard-index.html", context)
+
+
+BenefitsAdminSite.site_header = "Administrator"
+BenefitsAdminSite.site_title = "Cal-ITP Benefits - Administrator"
```

Comment:
@indexing What should the default title of the Admin app be?

This will set it so that each page is something like, `Log in | Cal-ITP Benefits - Administrator`

Reply:
We may want to set these 2 values in the templates themselves @machikoyasuda since `site_title` can be different across the pages of in-person enrollment and `site_header` is only used in the login page. So setting `site_title` using the `title` block and setting `site_header` directly in the `branding` block.